### PR TITLE
add kaggle icon

### LINKED ISSUE #427: Add Kaggle
## The icon I want is: Kaggle

* Name: Kaggle
* URL: https://www.kaggle.com/
* Link to icon: https://cdn4.iconfinder.com/data/icons/logos-and-brands/512/189_Kaggle_logo_logos-512.png

* Vector version of the logo: https://storage.googleapis.com/kaggle-competitions/kaggle/3136/media/kaggle-transparent.svg
* Official style guide or brand guide: https://www.kaggle.com/brand-guidelines

### PR DESCRIPTION
**Pull request by brunocerecetto**
resolve #427

### CODE REVIEW

File: images/svg/kaggle.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="..." role="img"
```
Comment (edent) [inline images/svg/kaggle.svg]: Please can you add the right Aria Label - then update the file size in the readme.

File: images/svg/kaggle.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+aria-label="..." role="img"
+viewBox="0 0 512 512"><rect
+width="512" height="512"
+rx="15%"
+fill="#fff"/><g transform="scale(5.5,5.5) translate(-40,-30)"><path fill="#20BEFF" d="M106,103c-0.06,0.3-0.3,0.4-0.8,0.4h-8c-0.5,0-0.9-0.2-1.3-0.6L82.746,86.028l-3.655,3.477v13c0,0.6-0.3,0.9-0.9,0.9h-6.152c-0.6,0-0.9-0.3-0.9-0.9V44c0-0.6,0.3-0.9,0.9-0.9h6.1c0.6,0,0.9,0.3,0.9,0.9v36l15.692-15.87c0.416-0.415,0.832-0.624,1.248-0.624h8.204c0.356,0,0.593,0.149,0.713,0.445c0.119,0.4,0.1,0.6-0.1,0.8L88,81,21.5C106,102,106,103,106,103"/></g></svg>
```
Comment (edent) [inline images/svg/kaggle.svg]: Is there a way to simplify this? The scale and translate can be applied directly to the path.

File: images/svg/kaggle.svg (hunk 1 above)
Comment (brunocerecetto) [reply images/svg/kaggle.svg]: done

**Comment by edent** [comment]
I'm granting you commit access to the Repo.  This is per the advice of [The Pull Request Hack](https://felixge.de/2013/03/11/the-pull-request-hack.html).

Please use this privilege wisely.

**Pull request merged**